packages/yourloops/pages/profile/profile.tsx

```diff
@@ -434,16 +394,47 @@ const ProfilePage = (props: ProfilePageProps): JSX.Element => {
                   <FormHelperText error>{t("profile-hcp-profession-helper-text")}</FormHelperText>
                 }
               </FormControl>
+
+              {user.settings?.country === "FR" &&
```

Comment:
Too many sub if() level in this "fake" DOM part
It's error prone for maintenance, it's hard to read

Reply:
I agree, this component is too big, I think it should be splitted into components. Maybe split the 3 sections (personnal info, credentials, preferences)

Reply:
Will be splitted in another PR, this one is already big enough !